app/server/idapiConfig.ts

```diff
@@ -0,0 +1,61 @@
+import * as AWS from "aws-sdk";
+import { GetObjectRequest } from "aws-sdk/clients/s3";
+import Raven from "raven";
+import { conf } from "./config";
+import { log } from "./log";
+
+export interface IdapiConfig {
+  host: string;
+  accessToken: string;
+}
+
+// tslint:disable-next-line:no-object-mutation
+process.env.AWS_PROFILE = "membership"; // TODO do this with CredentialProviderChain
+
+const s3 = new AWS.S3({
+  region: "eu-west-1"
+  // credentialProvider: new AWS.CredentialProviderChain([
```

Comment:
Is this going to be revisited in a subsequent PR or should it be removed?

Reply:
it will be - it works fine now by setting the env variable (and is the way _dot-components_ is doing it - just @adamnfish suggested using the `CredentialProviderChain ` explicitly - which I will get round to doing it soon